src/__spec_helper__/test-utils.js

```diff
@@ -364,6 +507,8 @@ export {
   carbonThemesJestTable,
   mockMatchMedia,
   testStyledSystemSpacing,
```

Comment:
I think that `testStyledSystemSpacing` can just be a function like 
```js
const testStyledSystemSpacing = (
  component,
  defaults,
  styleContainer,
  assertOpts
) => {
  testStyledSystemMargin(component, defaults, styleContainer, assertOpts);
  testStyledSystemPadding(component, defaults, styleContainer, assertOpts);
};
```

rather than duplicating the test logic

Reply:
🤦